test/unit/auth/credential.spec.ts

```diff
@@ -440,6 +440,56 @@ describe('Credential', () => {
       expect((getApplicationDefault())).to.be.an.instanceof(RefreshTokenCredential);
     });
 
+    it('should return a RefreshTokenCredential with FIREBASE_EMULATOR_CREDENTIALS', () => {
+      const firebaseEmulatorCred = {
+        client_id: "fakeclientid",
+        client_secret: "fakeclientsecret",
+        refresh_token: "fakerefreshtoken",
+        type: "authorized_user",
+      };
+      process.env.FIREBASE_EMULATOR_CREDENTIALS = JSON.stringify(firebaseEmulatorCred);
+
+      const defaultCred = getApplicationDefault();
+      expect(defaultCred).to.be.an.instanceof(RefreshTokenCredential);
+      expect(defaultCred).to.have.property('refreshToken').that.includes({ 
+        clientId: firebaseEmulatorCred.client_id,
+        clientSecret: firebaseEmulatorCred.client_secret,
+        refreshToken: firebaseEmulatorCred.refresh_token,
+      });
+    });
+
+    it('should throw if FIREBASE_EMULATOR_CREDENTIALS is invalid', () => {
+      process.env.FIREBASE_EMULATOR_CREDENTIALS = "{{}}";
```

Comment:
I believe you need that `after()` block to unset this. Otherwise all subsequent tests see this invalid value and fail.